scripts/data_source_etl.py

```diff
@@ -334,7 +334,6 @@ def main(config, make_attr=False):
         if 'programs' in config:
             for prog in config['programs']:
                 try:
-                    obj = Program.objects.get(name=prog['name'], owner=isb_superuser, active=True, is_public=True)
                     logger.info("[STATUS] Program {} found - skipping creation.".format(prog))
```

Comment:
I missed this change at some point--this breaks this code segment. It's meant to trigger an exception when the program isn't found. If the complaint was obj isn't used, the solution is only to remove the variable assignment, not the whole line.